frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -63,7 +63,11 @@ export default class ProfileLink extends Component {
       {
         title: t`Help`,
         icon: null,
-        link: MetabaseSettings.docsUrl(),
+        link:
+          isAdmin && MetabaseSettings.isPaidPlan()
```

Comment:
The logic is based on a comment here (Under `UTM Trackings` at `Non-admin` on the first row, I can't link to comment inside a table for some reason) https://www.notion.so/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977?d=8acba01bac36481c99da1b786f4f57e6